locust/run.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+###
+# Usage:
+#
+# Export values for the following env variables.
+#
+# Required:
+# BB_CLIENT_ID (application's client id)
+# BB_CLIENT_SECRET (application's client secret)
+# BB_SUB_DOMAIN (sub domain to test against, e.g., sandbox.bluebutton.cms.gov)
+#
+# Optional:
+# BB_NUM_BENES (number of synthetic benes, default: 5)
+# BB_LOAD_TEST_DURATION (number of seconds, default: 20)
+# BB_LOAD_TEST_HATCH_RATE (hatch rate for clients added per second, default: 1)
+# BB_LOAD_TEST_CONCURRENCY (how many clients make requests at once, default: 1)
+# BB_LOAD_TEST_MIN_WAIT (how many ms to wait between requests, lower bound, default: 1000)
+# BB_LOAD_TEST_MAX_WAIT (how many ms to wait between requests, upper bound, default: 5000)
+###
+
+set -e
+
+BB_LOAD_TEST_TYPE="${1:-BB_LOAD_TEST_TYPE}"
+
+if [ "$BB_LOAD_TEST_TYPE" != "eob" ] && [ "$BB_LOAD_TEST_TYPE" != "all" ]
+then
+  echo "Must specify a load test type (eob|all)" >&2
+  exit 1
+fi
+
+docker build -f ./Dockerfiles/Dockerfile.tkns -t bb_tkns .
+docker build -f ./Dockerfiles/Dockerfile.locust -t bb_locust .
+
+set +e
```

Comment:
Right now, `tkns` reliably fetches at least a few access tokens, but sometimes exits with code `> 0` due to a bug we haven't quite tracked down yet. I'm using `set +e` to ignore the exit code and let the script continue for the time being. We can/should tidy this up later.